x-pack/plugins/ml/server/models/data_recognizer/modules/sample_datasets_weblogs/ml/mean_bytes_by_file_extension.json

```diff
@@ -0,0 +1,38 @@
+
+{
+  "groups": [],
+  "description": "Kibana Sample Datasets Web Logs : detect anomalies in mean bytes",
+  "analysis_config" : {
+    "bucket_span": "1h",
+    "detectors": [
+      {
+        "detector_description": "Mean bytes transmitted",
+        "function": "mean",
+	"field_name": "bytes",
+	"partition_field_name": "extension.keyword"
+      }
+    ],
+    "influencers": ["extension.keyword", "clientip"]
+  },
+  "analysis_limits": {
+    "model_memory_limit": "10mb"
+  },
+  "data_description": {
+    "time_field": "timestamp"
+  },
+  "model_plot_config": {
+    "enabled": true
+  },
+  "custom_settings": {
+    "created_by": "ml-module-sample",
+    "custom_urls": [
+      {
+        "url_name": "Raw data",
+        "url_value": "kibana#/discover?_g=(time:(from:\u0027$earliest$\u0027,mode:absolute,to:\u0027$latest$\u0027))&_a=(index:\u002790943e30-9a47-11e8-b64d-95841ca0b247\u0027)" 
+	},
+     {
+	"url_name" : "Data dashboard",
+	"url_value" : "kibana#/dashboard/edf84fe0-e1a0-11e7-b6d5-4dc382ef7f5b?_g=(time:(from:\u0027$earliest$\u0027,mode:absolute,to:\u0027$latest$\u0027))&_a=(filters:!(),query:(language:kuery,query:\u0027\u0027))"
```

Comment:
IMO it would be worth passing the file extension to the target dashboard, in the filters section if possible.